flink-connector/build.gradle.kts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+plugins {
+  `maven-publish`
+  id("java")
+  id("idea")
+}
+
+repositories {
+  mavenCentral()
+}
+
+val flinkVersion: String = libs.versions.flink.get()
+val scalaVersion: String = project.properties["scalaVersion"] as? String ?: extra["defaultScalaVersion"].toString()
+
+dependencies {
+  implementation(project(":api"))
+  implementation(project(":common"))
+  implementation(project(":clients:client-java-runtime", configuration = "shadow"))
```

Comment:
`client-java-runtime` seems contains `api` and `common`

Reply:
I use `client-java` instead of `client-java-runtime` and will do the shadow in the `flink-connector-runtime`.

Reply:
It's confusing to use both `client-java` and `client-java-runtime`,  and we'd better place `implemetation` items together.

Reply:
I guess flink it will encounter https://github.com/datastrato/gravitino/issues/3441 if removing depondencies of `client-java` and `common` `api`

Reply:
@FANNG1 Yes. In addition, splitting integration testing into independent modules can also lead to concurrency issues in the testing environment.